wallet/db.c

```diff
@@ -376,12 +376,15 @@ static struct migration dbmigrations[] = {
 	{ "ALTER TABLE channels ADD feerate_base INTEGER;", NULL },
 	{ "ALTER TABLE channels ADD feerate_ppm INTEGER;", NULL },
 	{ NULL, migrate_pr2342_feerate_per_channel },
-	{ "ALTER TABLE channel_htlcs ADD received_time INTEGER", NULL },
-	{ "ALTER TABLE forwarded_payments ADD received_time INTEGER", NULL },
-	{ "ALTER TABLE forwarded_payments ADD resolved_time INTEGER", NULL },
+	{ "ALTER TABLE channel_htlcs ADD received_time INTEGER;", NULL },
+	{ "ALTER TABLE forwarded_payments ADD received_time INTEGER;", NULL },
+	{ "ALTER TABLE forwarded_payments ADD resolved_time INTEGER;", NULL },
```

Comment:
Please don't modify existing migrations. While this change may be harmless cosmetics, changing the migrations may end up corrupting deployed databases.

Reply:
Sorry, I'll take care, too